core/src/apps/cardano/layout/__init__.py

```diff
@@ -6,6 +6,7 @@
 from trezor.ui.text import Text
 from trezor.utils import chunks
 
+from apps.cardano.protocol_magics import protocol_magic_to_ui_string
```

Comment:
a thing i forgot previously:

please use relative imports for all `apps.cardano` imports, in all files under `apps/cardano`

i.e.:

```
from apps.common.confirm import (...)

from ..protocol_magics import to_ui_string
```

Reply:
Done in [here](https://github.com/trezor/trezor-firmware/pull/1096/commits/10305274dbc6f66e0e7007c0a7ecbc0fddbc1372) and [here](https://github.com/trezor/trezor-firmware/pull/1096/commits/7a2d04e20370f03fc47dd24f7e02286866b1ff0a).